app/src/models/merge.ts

```diff
@@ -36,3 +36,12 @@ export interface IMergeUnsupported {
 }
 
 export type MergeResult = IMergeSuccess | IMergeError | IMergeUnsupported
+
+export enum MergeSource {
+  BranchDropDown,
+  MergeIntoCurrentBranchMenuItem,
+  UpdateBranchMenuItem,
+  Compare,
```

Comment:
:art: could this be renamed to `CompareTab` - it feels less contextual at first glance compared to the others